tests/integration/backward_compatible/proxy/map_nearcache_test.py

```diff
@@ -108,3 +111,71 @@ def fill_map_and_near_cache(self, count=10):
         for k, v in six.iteritems(fill_content):
             self.map.get(k)
         return fill_content
+
+
+ENTRY_COUNT = 100
+
+
+class NonStopNearCacheTest(HazelcastTestCase):
+    def setUp(self):
+        rc = self.create_rc()
+        cluster = self.create_cluster(rc, self.read_cluster_config())
+        cluster.start_member()
+
+        def event_collector():
+            events = []
+
+            def collector(e):
+                if e == LifecycleState.DISCONNECTED:
+                    events.append(e)
+
+            collector.events = events
+            return collector
+
+        collector = event_collector()
+
+        client = HazelcastClient(
+            cluster_name=cluster.id,
+            reconnect_mode=ReconnectMode.ASYNC,
+            near_caches={"map": {}},
+            lifecycle_listeners=[collector],
+        )
+
+        map = client.get_map("map").blocking()
```

Comment:
This blocking/non-blocking does not exist in Java client right? or in java the default is blocking ?

Reply:
Yes, but it doesn't matter

Reply:
Yes to which question 😄  I guess to second one. I asked this just to learn as well. We have getAsync etc in the java client. I guess they are the non-blocking ones.

Reply:
The second, it exists for some methods as a `...Async` variant